include/hal/library/cryptlib.h

```diff
@@ -2890,6 +2890,22 @@ bool libspdm_x509_get_cert_from_cert_chain(uint8_t *cert_chain,
                                            const int32_t cert_index, uint8_t **cert,
                                            size_t *cert_length);
 
+/**
+ * Get one X509 total certificate chain len
+ *
+ * @param[in]      cert_chain       One or more ASN.1 DER-encoded X.509 certificates
+ *                                  where the first certificate is signed by the Root
+ *                                  Certificate or is the Root Cerificate itself. and
+ *                                  subsequent cerificate is signed by the preceding
+ *                                  cerificate.
+ * @param[in]      max_cert_chain_length   Max length of the certificate chain to parse
+ *
+ *
+ * @retval  cert_chain_len           If cert_chain_len == 0,failed; else, Success.
+ **/
+size_t libspdm_x509_get_cert_chain_len(uint8_t *cert_chain,
```

Comment:
This is (probably) no longer needed, or if it is it can be defined in `os_stub`. The size of the SPDM certificate chain is given in the `SET_CERTIFICATE` request.

Reply:
Thanks. 
 I change the code and use the follow code to get the actual cert_chain size in `libspdm_write_certificate_to_nvm`.

```
    spdm_cert_chain_t *cert_chain_header;
    size_t cert_chain_size;

    cert_chain_header = cert_chain;

    /*get actual cert_chain size*/
    cert_chain_size = cert_chain_header->length - sizeof(spdm_cert_chain_t) - root_cert_hash_size;
```